README.md

```diff
@@ -120,6 +120,16 @@ humanizeDuration(22140000, { conjunction: ' and ', serialComma: false })  // '6
 humanizeDuration(22141000, { conjunction: ' and ', serialComma: false })  // '6 hours, 9 minutes and 1 second'
 ```
 
+**maxDecimalPoints**
+
+Number that defines a maximal decimal points for float values.
+
+```js
+humanizeDuration(8123.456789) // 8.12 seconds
```

Comment:
Could you also add two more examples (probably at the end) showing:

1. what happens when `maxDecimalPoints` is greater than the number of decimal points that'd show up "normally"?
1. what happens when the value wouldn't have _any_ decimal points and `maxDecimalPoints` effectively does nothing?

Also, to be consistent with the other options, could you add spacing around the object definitions? So it should look like this:

```javascript
{ maxDecimalPoints: 3 }
```